Circuit/Components/VacuumTubes/Triode.cs

```diff
@@ -85,6 +90,19 @@ public class Triode : Component
         [Serialize, Category("Dempwolf-Zolzer")]
         public Quantity Ig0 { get { return ig0; } set { ig0 = value; NotifyChanged(nameof(Ig0)); } }
 
+        private double _cgp = 2.4e-12;
```

Comment:
These should be using `Quantity` with `Units.C`, and using a decimal constant (see https://github.com/dsharlet/LiveSPICE/blob/0d59564ea4ab6dc64c0e7e9d16dadb2d050e4ae4/Circuit/Components/Capacitor.cs#L15 for example)

Reply:
Done